lib/internal/loader/singleton.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
```

Comment:
It doesn't look like this commit is going to be used in this PR anymore. I suggest leaving it out for future PRs.